google-cloud-bigquery/src/main/java/com/google/cloud/bigquery/TableInfo.java

```diff
@@ -205,6 +211,7 @@ static class BuilderImpl extends Builder {
       this.labels = Labels.fromPb(tablePb.getLabels());
       this.requirePartitionFilter = tablePb.getRequirePartitionFilter();
       this.defaultCollation = tablePb.getDefaultCollation();
+      this.cloneDefinition = CloneDefinition.fromPb(tablePb.getCloneDefinition());
```

Comment:
Not sure if you want it here or elsewhere in fromPb, but don't you need to handle null cloneDefinition?

Reply:
Fixed